src/components/ListItem/DefaultListItem/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import styles from '@/components/ListItem/DefaultListItem/DefaultListItem.css';
+
+export interface Props<T> {
+  className?: string;
+  item: T;
+  getLabel: (item: T) => React.ReactNode;
+  size: string;
+  variablesClassName?: string;
+  getIcon?: (item: T, size: number) => React.ReactNode;
+  getSelectedMarker?: (item: T) => React.ReactNode;
+  getIsSelected?: (item: T) => boolean;
+}
+
+const DefaultListItem = <T extends {}>(props: Props<T>) => {
+  const {
+    className,
+    size,
+    getLabel,
+    getIcon,
+    getSelectedMarker,
+    item,
+    variablesClassName,
+    ...listItemProps
+  } = props;
+
+  const sizeSpanClass = `list-item--span-${size}`;
+
+  return (
+    <div
+      {...listItemProps}
+      className={classNames(className, styles['list-item'], variablesClassName)}
+      role='button'
+      aria-pressed='false'
+      tabIndex={0}
+    >
+      <div>{getIcon && (size === 'large' ? getIcon(item, 20) : getIcon(item, 13))}</div>
+      <span className={classNames(styles['list-item-span'], styles[sizeSpanClass])}>
+        {getLabel(item)}
+      </span>
+      <span>{getSelectedMarker && getSelectedMarker(item)}</span>
```

Comment:
this will always render even if no function is passed because of your default value